db/migrate/20221019102858_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Kindly change the name of the foreign key column name to `author` instead of `user`.

Reply:
@ShadyShawkat thanks for the review  :pray:  . I will make the corrections accordingly. I actually have some confusions that you may help me with. 

while generating the foreign keys to the comments, I used the following commands(same for the likes too)

$ rails generate migration AddPostRefToComments post:references
$ rails generate migration AddUserRefToComments user:references

The first one created the post_id column and the index in the comments while the second one did the same with the user_id. I think I need to change all the user_id s to author_id in the schema too because at this point I could not implement the $ rails db: migrate command(after pushing the repo) to update the schema or what do you recommend me? Also, do I need to create separate indexes, because I haven't used the second option in the screenshots?

![Screenshot from 2022-10-19 15-48-28](https://user-images.githubusercontent.com/75738563/196695655-b99fe84e-0ff2-4acf-92b9-da620575fe67.png)

Reply:
Hi @AbrahaKahsay if I understand yu correctly, the first option creates the index and foreign key as well.